river/command/output.zig

```diff
@@ -20,6 +20,7 @@ const mem = std.mem;
 
 const wlr = @import("wlroots");
 
+const flags = @import("flags");
```

Comment:
minor nitpick: flags here is a package and should be grouped with the other package import. Right now it is grouped with server, which is a global imported from a local sourcce file.

Reply:
Hope I understood correctly, I copied the grouping from `map.zig`, should be fixed.